pyClient/zeth/joinsplit.py

```diff
@@ -316,30 +394,117 @@ class ZethClient:
     """
     def __init__(
             self,
+            web3: Any,
             prover_client: ProverClient,
+            mk_tree_depth: int,
             mixer_instance: Any,
+            merkle_root: str,
             zksnark: IZKSnarkProvider):
         self._prover_client = prover_client
-        self._mixer_instance = mixer_instance
+        self.web3 = web3
         self._zksnark = zksnark
+        self.mixer_instance = mixer_instance
+        self.mk_tree_depth = mk_tree_depth
+        self.merkle_root = merkle_root
+
+    @staticmethod
+    def open(
+            web3: Any,
+            prover_client: ProverClient,
+            mk_tree_depth: int,
+            mixer_instance: Any,
+            zksnark: IZKSnarkProvider) -> ZethClient:
+        """
+        Create a client for an existing Zeth deployment.
+        """
+        return ZethClient(
+            web3,
+            prover_client,
+            mk_tree_depth,
+            mixer_instance,
+            contracts.get_merkle_root(mixer_instance).hex(),
+            zksnark)
+
+    @staticmethod
+    def deploy(
+            web3: Any,
+            prover_client: ProverClient,
+            mk_tree_depth: int,
+            deployer_eth_address: str,
+            zksnark: IZKSnarkProvider,
+            token_address: Optional[str] = None,
+            deploy_gas: Optional[EtherValue] = None) -> ZethClient:
```

Comment:
Why not pass it in Wei directly?

Reply:
`EthereValue` allows the caller to specify the value in any units, and handles the conversion from srting (e.g. from command line params) and ints in code.